core/src/utils/keyboard/keyboard-controller.ts

```diff
@@ -0,0 +1,45 @@
+import { win } from '@utils/window';
```

Comment:
We can't use relative imports in non-test code until https://github.com/ionic-team/stencil/issues/2815 is resolved.